src/vs/workbench/parts/debug/browser/debugEditorModelManager.ts

```diff
@@ -325,7 +325,7 @@ export class DebugEditorModelManager implements wbext.IWorkbenchContribution {
 
 	private static FOCUSED_STACK_FRAME_DECORATION: editorcommon.IModelDecorationOptions = {
 		isWholeLine: true,
-		className: 'debug-focussed-stack-frame-line',
+		className: 'debug-focused-stack-frame-line',
```

Comment:
Everything looks OK to me. Let's just wait for the builds to go green, and I'll merge it right after.  Thanks for this! :+1: